User matching query does not exist.
Traceback (most recent call last):
  File "E:\Django_Projects\Service-Request-Automation-Django\employeeInfo\views.py", line 110, in loginView
    user = User.objects.get(username=username)
  File "C:\Users\najmus\AppData\Roaming\Python\Python310\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\najmus\AppData\Roaming\Python\Python310\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
employeeInfo.models.User.DoesNotExist: User matching query does not exist.
User matching query does not exist.
2025-06-01 15:23:03.318300 - ERROR -  User matching query does not exist. 
